### DISCUSSION TITLE
Welcome to OnlineGo Discussions!

### DISCUSSION THREAD

**Discussion by acristescu** [acristescu/OnlineGo#83] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MathieuDreano** [comment]
Hello there, 
I'm a french developper, living in Nantes.
I am not an Android dev, but I'm willing to learn it. 
I play go, mostly on OGS, on a beginner level (~10kyu).
I'll use this project as a way to keep up with android framework.
Don't have much time as I just start a new job, but I'll do my part to contribute.
Feel free to talk, and maybe share a game sometime (Rakoon pseudoname).
Mathieu

**Reply by acristescu** [reply]
Hi @MathieuDreano ,

Welcome to the project. I feel this project is an interesting one to keep up to date with Android, just be careful that it has some obsolete parts still in there that I didn't get a chance to update. If you want the latest and greatest, look for the Game Screen, which is using some ultra-modern technologies: not only Compose which is becoming mainstream now but Jake Wharton's molecule for example.

Some of it is very old though, if you see a RecyclerView, run the other way, lol :)

If you want guidance, please ask and I'll do my best to answer, although it's rather hard to find the time for the project right now.
